maven-version-rules.xml

```diff
@@ -18,12 +18,12 @@
                 <ignoreVersion type="regex">.*(alpha|beta).*</ignoreVersion>
             </ignoreVersions>
         </rule>
-        <rule groupId="javax.xml.bind">
+        <rule groupId="jakarta.xml.bind">
             <ignoreVersions>
                 <ignoreVersion type="regex">.*-b[0-9]+.*</ignoreVersion>
             </ignoreVersions>
         </rule>
-        <rule groupId="org.glassfish" artifactId="javax.el">
+        <rule groupId="org.glassfish" artifactId="jakarta.el">
             <ignoreVersions>
                 <ignoreVersion type="regex">.*-b[0-9]+.*</ignoreVersion>
             </ignoreVersions>
```

Comment:
Please also change all locations in this file that have comment that never versions are held back because they require spring 6.x. And then run `mvn versions:display-property-updates` and bump pom.xml versions for those.

Reply:
Sure will do.

Reply:
Actually this was due to conflicting junit5 and junit platform commons versions. Working now.